docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 sphinx >= 2.1
 sphinx-autodoc-typehints
 sphinx-rtd-theme
+docutils==0.16
```

Comment:
https://stackoverflow.com/a/68008428

Reply:
The [last comment](https://stackoverflow.com/questions/67542699/readthedocs-sphinx-not-rendering-bullet-list-from-rst-file/68008428#comment125633564_68008428) on that post seems to indicate upgrading sphinx-rtd-theme and/or sphinx might resolve the issue. But I assume it's not a big deal either way.

Reply:
Yeah, I tried various upgrades unsuccessfully. docutils @ 0.16 was the only way I found to resolve it.